docs/sources/operators-guide/configure/setting-helm-ha-deduplication-consul/index.md

```diff
@@ -0,0 +1,138 @@
+---
+aliases:
+  - /docs/mimir/latest/operators-guide/configuring/setting-helm-ha-deduplication-consul/
+description: Learn how to configure Grafana Mimir Helm Chart to handle HA Prometheus server deduplication with Consul.
+menuTitle: Configuring high-availability deduplication with Consul
+title: Configuring Grafana Mimir Helm Chart for high-availability deduplication with Consul
+weight: 70
+---
+
+# Configuring mimir-distributed Helm Chart for high-availability deduplication with Consul
```

Comment:
```suggestion
# Configure high-availability deduplication with Consul
```
Match the title of in the metadata because it aligns with vetted practices.